lib/migration_data/active_record/migration.rb

```diff
@@ -15,6 +15,8 @@ def self.included(base)
         base.class_eval do
           def exec_migration_with_data(conn, direction)
             origin_exec_migration(conn, direction)
+            ::ActiveRecord::Base.connection.schema_cache.clear!
```

Comment:
We met this problem few times in past when same migration added column and the tried to fill it using ActiveRecord. New column wasn't in columns cache and it just crashed. 
It even happened few times when it were different migrations, run in the same rake process.
I think, though, that I can freely create it as separate pull request.
I just thought, that instead of forking this I could just only require the gem on needed environments, which would effectively skip data migrations on test env.